src/DriverTypes.hpp

```diff
@@ -274,7 +274,7 @@ struct AcousticConnection
  */
 struct DeviceSettings
 {
-    base::Time time;
+//    base::Time time;
```

Comment:
If you want to get rid of this line, _get rid of it_. Don't commit commented-out code.